src/components/SideBar/SideBar.js

```diff
@@ -0,0 +1,77 @@
+const SideBar = () => {
```

Comment:
I guess FilterContainer or FilterSidebar something containing Filter would be better name for this as it contains all of your Filters as it's not just a sidebar containing links to different pages.